documentation/content/en/books/handbook/wayland/_index.adoc

```diff
@@ -370,6 +370,11 @@ To start Hikari, use the following command:
 [[wayland-sway]]
 == The Sway Compositor
 
+[NOTE]
+====
+Make sure you have read this section 'How to Identify and Configure Drivers for Your Graphics Hardware'. otherwise it would give error "0 GPU detected".
```

Comment:
I'd make that a link, as explained in https://docs.freebsd.org/en/books/fdp-primer/asciidoctor-primer/#_links_to_the_same_file_or_to_another_file_in_the_same_book .